### ISSUE TITLE
Make Interactive Elements display more Human Readable

### ISSUE THREAD

**Issue by GustavoNazarioPerez** [hemkan/MotorEase#8]
### Description
**User Story**:  As a user, I should be able to understand what the mentioned interactive feature means.

After the feature outlined in issue #6 is implemented, users will be able to see what elements in the app and screenshot are labeled as interactive in the Accessibility Report. The format of these elements will follow what is outlined in the XML file that is produced by UiAutomator. This task aims at making those elements more human-readable as the XML files can be difficult to read, especially if one lacks familiarity with the structure.

**Ranking and Reasoning**: 
This is ranked 4th because chronologically, it is the next step after making the elements visible to the users via the Accessibility Report. Our ranking system is prioritized in the following way: errors > usability/function > UI. This would fall under usability/function as it enhances user experience and usability. 

### Dependencies
This task is dependent on #6. After #6 is completed we will be able to see all of the interactive elements in the `AccessibilityReport.txt` but in a structure resembling the XML file created by UiAutomator. The goal of this task is to translate this in a way that is easier to understand for the reader.

### Ideas for Fix
Since some of the team has experience with reading XML files from UiAutomator our idea is to parse the XML and provide things like the resource ID and location to the user in a more organized way than the view hierarchy currently structures it. Alongside this we were thinking of translating bounds to actual locations within the screenshot. For example: Say top right of the screen instead of [x, y] to make it easier for the user to locate the violating element.

### Timeline
Engineering Points/Effort: 2 points (2 full day's worth of work). This bug fix should be completed by Sprint 2 but is dependent on the completion of another task as mentioned in the Dependencies section.

### Acceptance Criteria
- Include a description of an interactive element in the Accessibility Report that is in natural language rather than XML format.
- Provide context to the element that is viewed as interactive based on the resource id and other parameters outlined in the XML file. For example: "This element is a text element".
